blocks/editable/index.js

```diff
@@ -220,8 +200,6 @@ export default class Editable extends Component {
 
 			if ( this.editor.dom.isEmpty( rootNode ) && this.props.onReplace ) {
 				this.props.onReplace( content );
-			} else {
-				this.splitContent( content );
```

Comment:
Why did this go?

Reply:
I don't know, I recall removing it explicitly for a reason but totally forgot why. I'm bringing it back.